gating/update_dependencies/run

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash -xe
+
+# Update OSA SHA to head of stable/ocata
+
+# These var must be set per branch of RPC-Openstack
+osa_branch=stable/ocata
+rpco_branch=master
+
+# Env vars injected by Jenkins:
+WORKSPACE="${WORKSPACE:-$PWD}"
+
+# Note for local testing: this script expects to be excecuted from the root
+# of an rpco clone, checkout at master.
+
+# Get current head of stable/ocata
+git clone "https://github.com/openstack/openstack-ansible" ${WORKSPACE}/oa
+pushd ${WORKSPACE}/oa
+  git checkout $osa_branch
+  osa_sha="$(git log -n 1 --format=%H)"
+popd
+
+# Insert current SHA into functions.sh
+sed -i '/OSA_RELEASE:-/ c\
+export OSA_RELEASE=${OSA_RELEASE:-"'${osa_sha}'"}' scripts/functions.sh
+
+
+# Update rpc_release. The mainline branch should always be one version
+# ahead of the rc branch, as the next rc branch will be cut from mainline
+# post release.
+
+# This update script will only ever run against a mainine branch, as
+# dependencies should not change once an RC branch has been cut.
+
+# Get RC branch version
+rc_branch="${rpco_branch}-rc"
+git fetch origin
+git show origin/${rc_branch} || {
+  echo "Rc-branch ${rc_branch} not found, skipping rpc_release version bump."
```

Comment:
nit:
``RC branch`` instead of ``Rc-branch``

Reply:
Done